### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by reyang**
As discussed in [the meeting today](https://docs.google.com/document/d/10xG7DNKWRhxNmFGt3yYd3980a9uwS8lMl2LvQL3VNK8/edit).

I'll create the tag once this PR got merged.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+## Changes
+
+Please provide a brief description of the changes here.
+
+Note: if the PR is touching an area that is not listed in the [existing areas](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/README.md), or the area does not have sufficient [domain experts coverage](https://github.com/open-telemetry/semantic-conventions/blob/main/.github/CODEOWNERS), the PR might be tagged as [experts needed](https://github.com/open-telemetry/semantic-conventions/labels/experts%20needed) and move slowly until experts are identified.
+
+## Merge requirement checklist
+
+* [ ] [CONTRIBUTING](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
+* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial changes.
```
Comment (trask) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: > Appropriate `CHANGELOG.md` files

I'm not sure what this means?

**Review by trask — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (reyang) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: How about this?

`CHANGELOG.md` updated for non-trivial changes.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Fixes #
+
+## Changes
+
+Please provide a brief description of the changes here.
+
+Note: if the PR is touching an area that is not listed in the [existing areas](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/README.md), or the area does not have sufficient [domain experts coverage](https://github.com/open-telemetry/semantic-conventions/blob/main/.github/CODEOWNERS), the PR might be tagged as [experts needed](https://github.com/open-telemetry/semantic-conventions/labels/experts%20needed) and move slowly until experts are identified.
+
+## Merge requirement checklist
```
Comment (jsuereth) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Can you add `schema-next.yaml updated with changes to existing conventions` as a checklist item?

**Review by jsuereth — APPROVED** [review]
Good first start.

We may want to point folks at how to collect expert groups and kick of new areas in semconv as well.

**Review by arminru — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (reyang) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: Done

**Pull request merged**